packages/fxa-settings/src/lib/utilities.ts

```diff
@@ -35,6 +36,16 @@ export function deepMerge(
   return deepMerge(target, ...sources);
 }
 
+export function getSearchParams(paramKeys: string[], locationHref: string) {
```

Comment:
I think this is going to be the preferred way to grab parameters moving forward.

I'm actually uncertain why we didn't do it this way when we originally made `searchParams` a few years ago - `new URL` was available in ES6, and 1) is nicer than our custom `splitEncodedParams` function etc., and 2) **in Storybook, we can modify `location.href` but we cannot modify `location.search`** since it's read-only.